### PR TITLE
Fix upstream issue 1520.

### PR DESCRIPTION
**Pull request by jfg956**
Fix upstream issue 1520.
https://github.com/vitessio/website/issues/1520

At the same time, consolidate "cp -r" in a single line.

Also, at the same time, allow variable binary path.

I would suggest squash-merge to put all this in a single commit (sorry, I did many commits because I did this change via the GitHub UI and I do not know how to put many changes in a single commit).

### CODE REVIEW

File: content/en/docs/15.0/get-started/local.md
Comment (mattlord) [inline content/en/docs/15.0/get-started/local.md:1]: The reorganization of the examples code was done in v16.0. We should not change the 15.0 doc page.

**Review by mattlord — CHANGES_REQUESTED** [review]
We should remove the 15.0 changes, but otherwise it LGTM. Thanks, @jfg956!

File: content/en/docs/15.0/get-started/local.md
Comment (jfg956) [reply content/en/docs/15.0/get-started/local.md:1]: Are you sure Matt, this is not what I see in {1}...

{1}: https://vitess.io/docs/15.0/get-started/local/

... nor in below...
```
vitess-15.0.3-652b0da]$ test -e examples/local && echo it is there
it is there
```

...from what I see, only `env.sh` has been moved from 15 to 16...
```
vitess_bins]$ test -e vitess-15.0.3-652b0da/examples/local/env.sh && echo it is there
it is there
vitess_bins]$ test -e vitess-15.0.3-652b0da/examples/common/env.sh && echo it is there || echo it is NOT there
it is NOT there
vitess_bins]$ test -e vitess-16.0.2-6076fed/examples/local/env.sh && echo it is there || echo it is NOT there
it is NOT there
vitess_bins]$ test -e vitess-16.0.2-6076fed/examples/common/env.sh && echo it is there || echo it is NOT there
it is there
```

...and my changes are working for v15, I tested see results below.
```
~]$ version=15.0.3; hash=652b0da
~]$ f=vitess-${version}-${hash}.tar.gz
~]$ dl=~/vitess  # dl: directory link
~]$ test -h $dl && rm $dl
~]$ db=~/vitess_bins/vitess-${version}-${hash}  # db: directory binaries.
~]$ test -e $db || ( cd ~/vitess_bins; tar -xzf $f; )
~]$ ln -s $db $dl
~]$ vitess_path=$dl
~]$ mkdir ~/my-vitess-example
~]$ cp -r ${vitess_path}/{examples,web} ~/my-vitess-example
~]$ cd ~/my-vitess-example/examples/local
local]$ ./101_initial_cluster.sh
[...]
```

Comment (mattlord) [reply content/en/docs/15.0/get-started/local.md:1]:
```
❯ pwd; git checkout release-15.0; git pull; ls -l ../common
/Users/matt/git/vitess/examples/local
Already on 'release-15.0'
Your branch is up to date with 'origin/release-15.0'.
Already up to date.
total 40
-rw-r--r-x  1 matt  staff  794 Oct 18  2022 insert_commerce_data.sql
-rw-r--r-x  1 matt  staff  162 Aug  1  2022 select_commerce_data.sql
-rw-r--r-x  1 matt  staff  129 Aug  1  2022 select_customer-80_data.sql
-rw-r--r-x  1 matt  staff  125 Aug  1  2022 select_customer0_data.sql
-rw-r--r-x  1 matt  staff  129 Aug  1  2022 select_customer80-_data.sql
```

This work was done in https://github.com/vitessio/vitess/pull/12239 which was only backported to v16.0.

Comment (mattlord) [reply content/en/docs/15.0/get-started/local.md:1]: Hmm, your noted changes would still be good though. Sorry! We did still have the other subdirs (especially web) that did need to be copied. 

So you're right, we do want this. 🙂

**Review by mattlord — APPROVED** [review]
Thank you! ❤️

**Pull request merged**